projects/sdk/src/lib/silo.ts

```diff
@@ -91,6 +93,7 @@ export class Silo {
     Silo.sdk = sdk;
     this.depositBuilder = new DepositBuilder(sdk);
     this.withdraw = new Withdraw(sdk);
+    this.claim = new Claim(sdk);
```

Comment:
Expanding on the `sdk.silo.claim.claim` note, feels strange to me to have `depositBuilder` (with "builder") but `withdraw` and `claim`. Understand though that the DepositBuilder is doing something functionally different, just thinking about this from an interface perspective.

Reply:
Yeah, it is weird, and it's something that bugged me too. The reason for depositBuilder, and swapBuilder, is to optimize in the UI. It gives you an object that has route calculated and allows you to play with the amounts w/o rebuilding the whole thing. We can discuss if there's way to improve this or if you think it's over-optimized.